lib/contents.php

```diff
@@ -53,7 +53,7 @@ function getContents($url, $header = array(), $opts = array()){
 	$cache->setParameters($params);
 
 	// Use file_get_contents if in CLI mode with no root certificates defined
-	if(php_sapi_name() === 'cli' && ini_get('curl.cainfo') === '') {
+	if(php_sapi_name() === 'cli' && (ini_get('curl.cainfo') === '' || ini_get('curl.cainfo') === false)) {
```

Comment:
This is a good spot for using [`empty`](http://php.net/manual/en/function.empty.php#refsect1-function.empty-returnvalues) because it covers all cases (same for the condition above).

```suggestion
	if(php_sapi_name() === 'cli' && empty(ini_get('curl.cainfo'))) {
```

Reply:
Yes, although there's a very small possibility that they use the filename "0" for the `curl.cainfo`.